swarm/src/lib.rs

```diff
@@ -1581,7 +1581,7 @@ pub enum DialError {
     /// has been reached.
     ConnectionLimit(ConnectionLimit),
     /// The peer being dialed is the local peer and thus the dial was aborted.
```

Comment:
These docs need updating now because we are now also using this variant for inbound connections.

Reply:
But this is `DialError`